libs/testing/e2e/src/lib/utils/pageHelpers.ts

```diff
@@ -0,0 +1,2 @@
+export const getInputByName = (name: string) => `input[name="${name}"]`
+export const getTextareaByName = (name: string) => `textarea[name="${name}"]`
```

Comment:
If unused, remove. Change where used to use `page.getByRole`

Reply:
Addressing in https://github.com/island-is/island.is/pull/16816